README.rst

```diff
@@ -94,7 +96,8 @@ written in a simple XML language.  Note this is not an XML format we are
 proposing as a standard.
 
 Once you install your XBlock into your virtualenv, the workbench will
-automatically display its scenarios for you to experiment with.
+automatically display its scenarios for you to experiment with. You don't have to use anything more complicated than pip to install the package:
```

Comment:
Also, looks like I was wrong here: use one colon and add a blank line, and the rst editor will be happier.